libpkg/scripts.c

```diff
@@ -198,6 +198,9 @@ pkg_script_run(struct pkg * const pkg, pkg_script type)
 			}
 
 			if (WEXITSTATUS(pstat) != 0) {
+				if (WEXITSTATUS(pstat) == 3)
```

Comment:
Wouldn't it be better to just change line 200 from:

`if (WEXITSTATUS(pstat) != 0)`

to

`if (WEXITSTATUS(pstat) != 0 || WEXITSTATUS(pstat) != 3)`